controllers/roverController.ts

```
@@ -0,0 +1,17 @@
+import { Request, Response } from "express"
+import { getRoverList } from "../clients/nasaClient"
+import {Rover} from "../models/roverData"
+
+export const RoverController = {
+    Index: async (req: Request, res: Response) => {
+        try {
+            const roverData: Rover[] | void = await getRoverList()
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)